### PR TITLE
Add modal Project etails, Dectop and Mobile

### PR DESCRIPTION
**Pull request by WillyStorm**
Add modals popup, project details page and mobile details.

### CODE REVIEW

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @WillyStorm, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from all members in your pair programming group. Kindly make sure that all the members contribute to the repo before submitting it for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const pdetails = {
+    title: 'Project name goes here',
+    text1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    text2: 'Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    slideshow: './images/slideshow.svg',
+    languages: {
+    lan1: 'HTML/CSS',
+    lan2: 'Ruby on Rails',
+    lan3: 'JavaScript',
+    },
```
Comment (sja-thedude) [inline modal.js]: - [ ] Kindly add the modal popup for all the projects shown in the video attached. Make sure to add a modal popup to all buttons named "See this project".
- [ ] Kindly add a link to the live version, and link to the project source code to the objects array.

https://user-images.githubusercontent.com/70269507/163392941-bc4dbf09-38a3-410f-b683-f985842f38bc.mov

File: index.html [hunk 2]
```diff
@@ -222,5 +223,31 @@ <h2>Get started</h2>
...
+                                </div>
+                                <div class="flex between">
+                                    <button type="button" class="inverse"><span><img src="images/lefty.png" alt="previous"></span>Previous project</button>
+                                    <button type="button" class="inverse end">Next project<span><img src="images/righty.png" alt="previous"></span></button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <!--End of Desctop Modal content -->
+                </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly make sure that the projects section and the popup are created dynamically with the information from the projects array. This requires changes in existing code that are already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @WillyStorm, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job in working on the popup.
✔️ Professional Readme

## Required 
- [ ] Kindly make sure to setup correct linters with js. Kindly add html-css-js linters from [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @WillyStorm, This is @sja-thedude your code reviewer.

Great work on making the changes requested  👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great job on implementing all the changes asked!

 ## Required Changes ♻️
- [ ] Kindly make sure to update your .github/workflows -> Linters.yml file and update it according to the html-css-js -> linters.yml from [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @WillyStorm, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966


 ## Highlights
✅ Popup works as expected
✅ Design follows Figma template

 ## Required Changes ♻️
- [x] Kindly set up your [JavaScript linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) as requested by your previous reviewer. This issue probably comes from the `linters.yml` file. Kindly make sure the Eslint linters checks show up on GitHub and they should pass before requesting a review 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team 👋🏿 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**